### DISCUSSION TITLE
Lesson 9 import problem "@chainlink/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol" not found: File not found

### DISCUSSION THREAD

**Discussion by aboronilov** [Cyfrin/foundry-full-course-cu#1638] [Q&A]
After adding a subscription ID I faced a problem ```Source "@chainlink/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol" not found: File not found. Searched the following locations: "".solidity(6275)```. Before subscription this line actually worked. And the file indeed exists

```solidity
// SPDX-License-Identifier: MIT

pragma solidity ^0.8.18;

import {Script} from "lib/forge-std/src/Script.sol";
import {LinkToken} from "test/mocks/LinkToken.sol";
import {VRFCoordinatorV2Mock} from "@chainlink/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol";
```
![sol](https://github.com/Cyfrin/foundry-full-course-f23/assets/82841854/c616f8ad-98c5-4a53-9c41-e5ba8b2aeab1)

What is more weird, the same line of code is working perfectly in another file **Interactions.s.sol**
```solidity
// SPDX-License-Identifier: MIT

pragma solidity ^0.8.18;

import {Script, console} from "lib/forge-std/src/Script.sol";
import {HelperConfig} from "script/HelperConfig.s.sol";
import {VRFCoordinatorV2Mock} from "@chainlink/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol";
```

My **foundry.toml**
```solidity
remappings = [
    "@chainlink/contracts/=lib/chainlink-brownie-contracts/contracts/",
    "@solmate=lib/solmate/src/",
]
```

**Comment by EngrPips** [comment]
Hello @aboronilov, After carefully reviewing the import path, I realized you mistakenly used the alphabet `O` instead of the number `0` in the file where the import isn't working. Correct that, and let's see if it works.

**Reply by drained69** [reply]
Actually looks like `0` lol

**Reply by drained69** [reply]
The screenshot made it look like `O` I think but if you look above you would confirm that it is `0`. He also says the file exist maybe he can push to github, I can help take a look here.

**Reply by EngrPips** [reply]
Oh, i see. @aboronilov did you figure it out already ?

**Reply by aboronilov** [reply]
@EngrPips no that's not the issue - because I copy and pasted that line of code from **Interactions.s.sol** where it is actually working. What is more weird, after reopening the project in VS code now this line gives no issues - the code compiles. I nave no idea what happened

**Reply by EngrPips** [reply]
I am glad it works now. vscode, at times vscode takes time to load entirely and sometimes glitches. That might have been the cause

**Reply by aboronilov** [reply]
@EngrPips  anyway thank you for your help

**Reply by EngrPips** [reply]
Anytime friend, Keep crushing it.
